tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -299,16 +305,26 @@ def initial_mirror_config(self, mirror2, poolname, imagename, **kw):
         if kw.get("image_feature"):
             image_feature = kw.get("image_feature")
             self.image_feature_enable(imagespec=imagespec, image_feature=image_feature)
+
         if kw.get("mode"):
             kw["peer_mode"] = kw.get("peer_mode", "bootstrap")
             kw["rbd_client"] = kw.get("rbd_client", "client.admin")
+            if kw.get("mode") == "image":
```

Comment:
can you please add some comments saying why are you setting `wait_for_status` to false?

Reply:
@sunilangadi2 Do you want me to add comments in the module or in this PR ?
If in module, I would add in config_mirror method, is that okay?

Reply:
yes please add it in the module method

Reply:
@sunilangadi2 done
Please check